src/main/features/response/views/how-much-owed.njk

```diff
@@ -0,0 +1,32 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, textInput, saveAndContinueButton, errorSummary, textArea, poundInput %}
+
+{% set title = t('Money claim') %}
+{% set heading = t('How much money do you believe you owe?') %}
+
+{% set claimantFullName = claim.claimData.claimant.name %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <form novalidate method="post">
+        {{ csrfProtection(csrf) }}
+        <p>{{ t('The amount being claimed is') }} £{{ amount }}</p>
```

Comment:
Please use the `numeral` filter to format money, see `src/main/features/claim/views/check-and-send.njk` for reference.